WordPressVIPMinimum/Sniffs/AbstractVariableRestrictionsSniff.php

```diff
@@ -58,7 +58,7 @@ abstract class AbstractVariableRestrictionsSniff extends Sniff {
 	/**
 	 * Returns an array of tokens this test wants to listen for.
 	 *
-	 * @return array
+	 * @return array<int|string>
```

Comment:
When a non-PHP native token constant is used.

The value of PHP native token constants are always integers.
The value of the custom PHPCS native token constants are strings, however.

So returning `T_ENUM` may be either an `int` or a `string` depending on the PHP version used to run PHPCS.
As of PHP 8.1 it would be an `int`, but for PHP < 8.1, the token is polyfilled by PHPCS, so it would be a `string`.

Does that help ?

Reply:
Perfectly!